src/controllers/authController.js

```diff
@@ -55,8 +103,16 @@ const login = async (req, res) => {
 };
 
 //logout endpoint
-
 const logout = async (req, res) => {
+  /*
+    #swagger.summary = 'Logout a user'
+    #swagger.description = 'This endpoint checks for a signed JWT token in an HTTP-only cookie and clears it. It checks the `token` field in the cookie.'
+    #swagger.responses[200] = {
+        description: 'User logged out successfully',
+        schema: { type: 'object', properties: { msg: { type: 'string' } } }
```

Comment:
this is showing:

![](https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/9094098/31ccac3d-edcb-4939-97d7-9a0b33c975e3)

for swagger-autogen we want to define the schema literally as it will be shown:

```suggestion
        schema: { msg: 'user logged out!' }
```


https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/9094098/4d8bb1a8-a60d-4e81-a9f4-3291cd32f360